composer.json

```diff
@@ -10,13 +10,14 @@
     "squizlabs/php_codesniffer": "^3.3",
     "wp-coding-standards/wpcs": "^1",
     "phpcompatibility/php-compatibility": "^9",
-    "dealerdirect/phpcodesniffer-composer-installer": "0.4.4"
+    "dealerdirect/phpcodesniffer-composer-installer": "0.4.4",
+    "johnpbloch/wordpress-core": "^5.2"
   },
   "require": {
     "php": ">=5.4",
     "google/apiclient": "^2.0",
     "guzzlehttp/guzzle": "~5.3.3",
-    "johnpbloch/wordpress-core": "^5.1"
+    "wordpress/core-implementation": "*"
```

Comment:
@TangRufus Why is it necessary to add this, especially why in `require` and not `require-dev`?

Reply:
The implementation package makes it package-agnostic. Dev packages are needed during development of the package itself, not when you want to load it as part of a composer project. So it makes sense to me.

Reply:
I‘d probably change the version constraint though to meet the actual requirements.

Reply:
`wordpress/core-implementation` seems to be something that the `johnpbloch/wordpress-core` repository introduced and not an official/documented standard. I doubt there will be an official standard unless WordPress itself adopts Composer but I am 👎 on adding this.

It forces and Composer projects that use Site Kit to also have a `provide` entry for `wordpress/core-implementation`. I am able to add it to the WordPress package my project consumes but some folks may not be in control of the WordPress core package they are consuming.

Essentially, I don't think Site Kit should for an unofficial policy on users.

Reply:
Then I suppose we can remove that line altogether, or move the existing one to dev requirements. The package type would be enough to indicate the environment I think.

Reply:
`wordpress/core-implementation` removed

Reply:
Thanks for the feedback. Let's track the `wordpress/core-implementation` for the future, whether we should add it at some point.